packages/client/src/object/fetchPage.ts

```diff
@@ -205,12 +229,46 @@ export async function fetchPageInternal<
   args: FetchPageArgs<Q, L, R, A> = {},
 ): FetchPageResult<Q, L, R> {
   if (objectType.type === "interface") {
-    return await fetchInterfacePage(client, objectType, args, objectSet) as any; // fixme
+    return await fetchInterfacePage(
+      client,
+      objectType,
+      args,
+      objectSet,
+    ) as any; // fixme
   } else {
     return await fetchObjectPage(client, objectType, args, objectSet) as any; // fixme
   }
 }
 
+/** @internal */
+export async function fetchPageWithErrorsInternal<
```

Comment:
Do we need these extra methods? Could fetchPageOrError() just call fetchPage but with the wrapResult call?

Reply:
We definitely do not, I initially kept them separate because of all the error handling stuff we had before so wasn't sure if I'd need to do some special stuff for interfaces, but without all that can totally just wrap the `fetchPage`